fastlane/Fastfile

```diff
@@ -0,0 +1,28 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+#   update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+  desc "Push a new beta build to TestFlight"
+  lane :beta do
+    setup_ci(provider: "travis")
+    match(readonly: true)
+    number = sh('date "+%Y.%m.%d.%H%M"')
+    increment_build_number(xcodeproj: "COVIDWatch.xcodeproj", build_number: number)
```

Comment:
Sets the app's build number to the current date and time, so something like: `2020.4.14.2134`